devfiles/quarkus/devfile.yaml

```diff
@@ -15,7 +15,7 @@ components:
     id: redhat/quarkus-java8/latest
   -
     type: dockerimage
-    alias: maven
+    alias: centos-quarkus-maven
```

Comment:
Why centos?

Reply:
because it is the original name of the image being used. I would like to emphasis the fact that we are based on containers and although we are patching it we are using https://quay.io/repository/quarkus/centos-quarkus-maven

Reply:
@ericwill ^^